suite-native/module-home/src/screens/HomeScreen/components/PortfolioGraphHeader.tsx

```diff
@@ -60,31 +60,33 @@ export const PortfolioGraphHeader = () => {
 
     return (
         <Box>
-            <VStack spacing="extraSmall" alignItems="center">
-                {!isLoading && (
-                    <>
-                        <Box justifyContent="center" alignItems="center" style={{ width: '100%' }}>
-                            <Balance />
-                        </Box>
-                        <HStack alignItems="center">
-                            {firstGraphPoint ? (
-                                <>
-                                    <GraphDateFormatter
-                                        firstPointDate={firstGraphPoint.date}
-                                        selectedPointAtom={selectedPointAtom}
-                                    />
-                                    <PriceChangeIndicator
-                                        hasPriceIncreasedAtom={hasPriceIncreasedAtom}
-                                        percentageChangeAtom={percentageChangeAtom}
-                                    />
-                                </>
-                            ) : (
-                                <Text>{' ' /* just placeholder to avoid layout shift */}</Text>
-                            )}
-                        </HStack>
-                    </>
-                )}
-            </VStack>
+            {!isLoading && (
```

Comment:
Won't this be as flaky as the previous version? The view with the specific `testID` will appear only after the discovery finishes, which is even later than the original check of discovered BTC account. 

Did you consider this? IMO it would be enough to check that discovery at least started.

Reply:
[1d22544](https://github.com/trezor/trezor-suite/pull/13158/commits/1d22544511330f1f7da87d080ea90aa9629f9ffe)